tests/python/roundtrips/image/main.py

```diff
@@ -30,7 +30,7 @@ def main() -> None:
 
     rr.log("image", rr.Image(image))
 
-    # 4x5 mono image. Pixel = x * y * 123.4
+    # 5x4 mono image. Pixel = x * y * 123.4
     image = np.zeros((4, 5), dtype=np.float16)
```

Comment:
I don't think we're doing ourselves any favors here by not naming the dimensions

```suggestion
    # h=4, w=5 mono image. Pixel = x * y * 123.4
    image = np.zeros((4, 5), dtype=np.float16)
```